### PR TITLE
Added Organization Popups

### LINKED ISSUE #337: Add popups to Organizations page
The goal is to add new popups suitable for the Organizations page. We will be starting with following popups:-
1. Community Followers popup
2. Organization Edit popup.

The designs will be same as those on Figma, with change in the data fields.

### PR DESCRIPTION
**Pull request by theprover97**
I have added the Edit Organization and Followers popups for Organization. It fixes #337 .

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@theprover97 Please add some gif or images of the UI

**Comment by theprover97** [comment]
Screenshot from 2020-04-02 21-56-28

**Comment by theprover97** [comment]
Screenshot from 2020-04-02 21-56-39

**Comment by tanujvyas10** [comment]
@Rupeshiya This PR also contains the pop-up for add the organization

> I have added the Edit Organization and Followers popups for Organization. It fixes #337 .

please clear a small doubt, if u added pop-up for edit organization then why your screenshots is saying  **New organization**?

File: src/user/organization/popups/EditOrg.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      });
+      
+    return (
+      <div>
+        <Dialog
+        {...this.props}
+        size="sm"
+        aria-labelledby="customized-dialog-title">
+          <DialogTitle> 
+            <span className="title">New Organization</span>
```
Comment (tanujvyas10) [inline src/user/organization/popups/EditOrg.js:47]: @theprover97 if its your pop-up for edit organization then why is it saying new Organization?

**Comment by Rupeshiya** [comment]
@tanujvyas10 I am going to close that issue for new organizations, as we can use this same popup to add organization as well.
Hope you don't mind!

**Comment by tanujvyas10** [comment]
> @tanujvyas10 I am going to close that issue for new organizations, as we can use this same popup to add organization as well.
> Hope you don't mind!

sure better to reduce redundancy
P.S. the following pop-up is of add-organization but mentioned as edit organization
@theprover97 you can make it the PR for add-organization pop-up

**Comment by theprover97** [comment]
Sure @Rupeshiya , I will keep that in mind. Also @tanujvyas10 thanks for your review!

**Comment by tanujvyas10** [comment]
@theprover97 your welcome buddy,  a little change in buttons is required i believe 
Please make them Update and discard respectively(since its edit so button labels will look more relevant ) and please squash your commits that's all

**Pull request merged**